src/System.Collections.NonGeneric/src/ApiCompatBaseline.net46.txt

```diff
@@ -0,0 +1,11 @@
+TypesMustExist : Type 'System.Collections.ArrayList' does not exist in the implementation but it does exist in the contract.
```

Comment:
@ericstj , @weshaggard : Yes, the command generated was correct. The .csproj didn't contain **<IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>** and the output dll didn't contain any typeforwards.
I will remove the baseline file as adding this line to the .csproj should fix it.